test/test_cuda/test_main_func.py

```diff
@@ -212,7 +212,6 @@ def test_low_cpu_mem_usage(self):
             group_size=group_size,
             iters=2,
             seqlen=10,
-            dataset=self.llm_dataloader,
```

Comment:
May I know what's the real reason that causes the ut fail and why change this could fix it.